tools/vp20compiler/config.h

```diff
@@ -6,10 +6,10 @@
 
 /** For GL_NV_vertex_program */
 /*@{*/
-#define MAX_NV_VERTEX_PROGRAM_INSTRUCTIONS 128
-#define MAX_NV_VERTEX_PROGRAM_TEMPS         12
-#define MAX_NV_VERTEX_PROGRAM_PARAMS        96
-#define MAX_NV_VERTEX_PROGRAM_INPUTS        16
-#define MAX_NV_VERTEX_PROGRAM_OUTPUTS       15
+#define MAX_NV_VERTEX_PROGRAM_INSTRUCTIONS 136 // FIXME: Xbox has 136 slots?
+#define MAX_NV_VERTEX_PROGRAM_TEMPS         13 // FIXME: Xbox has 13 regs, where R12 is oPos
+#define MAX_NV_VERTEX_PROGRAM_PARAMS       192 // FIXME: Xbox has 192 regs [-96 to 95]
+#define MAX_NV_VERTEX_PROGRAM_INPUTS        16 // FIXME: Xbox has 16 inputs?!
+#define MAX_NV_VERTEX_PROGRAM_OUTPUTS       11 // FIXME: Xbox has 16 outputs?!
```

Comment:
This will be fixed in main.c, the parser will keep using (11+4). 11 nv2a regs + tex4 to tex7 to fulfil GL spec